sharding-core/sharding-core-rewrite/src/main/java/org/apache/shardingsphere/core/rewrite/feature/encrypt/EncryptCondition.java

```diff
@@ -56,18 +58,20 @@
     
     private final Map<Integer, Object> positionValueMap = new LinkedHashMap<>();
     
-    public EncryptCondition(final String columnName, final String tableName, final int startIndex, final int stopIndex, final ExpressionSegment expressionSegment) {
+    public EncryptCondition(final String columnName, final String tableName, final String tableOwner, final int startIndex, final int stopIndex, final ExpressionSegment expressionSegment) {
         this.columnName = columnName;
         this.tableName = tableName;
+        this.tableOwner = tableOwner;
         this.startIndex = startIndex;
         this.stopIndex = stopIndex;
         operator = ShardingOperator.EQUAL;
         putPositionMap(0, expressionSegment);
     }
     
-    public EncryptCondition(final String columnName, final String tableName, final int startIndex, final int stopIndex, final List<ExpressionSegment> expressionSegments) {
```

Comment:
Do you think it is better to store tableAlias instead of tableOwner, for people will puzzle by tableName and tableOwner from this list of parameters. Therefore if tableAlias is nulll, we can pass tableName to `EncryptPredicateToken` as a tableOwner, otherwise tableAlias will be tableOwner. What' your opinion?

Reply:
Actually the variable `tableOwner` and `tableAlias` have the same meaning, I can rename `tableOwner` to `tableAlias`.  But if I change the logic as you said, 
the SQL
`select * from table where pwd = 1` 
will be rewritten as 
`select * from table where table.cipher  = xxx`
This result SQL is right?